platform/ios/MGLMapboxEvents.m

```diff
@@ -136,7 +195,26 @@ - (instancetype) init {
             
             [[NSUserDefaults standardUserDefaults] registerDefaults:defaultsToRegister];
         }
+    }
+}
+
++ (BOOL)isEnabled {
+    return ([[NSUserDefaults standardUserDefaults] boolForKey:@"MGLMapboxMetricsEnabled"] &&
+            [[NSUserDefaults standardUserDefaults] integerForKey:@"MGLMapboxAccountType"] == 0);
```

Comment:
Can we `assert()` here to ensure that the values are set and not just check their values? Because we are dealing with `BOOL` and `NSInteger`, I could see future potential traps around these being `nil` in some circumstances. If we can ensure that they are non-`nil` we can be proactive about those.